src/hooks/useDisabledStyle.tsx

```diff
@@ -0,0 +1,11 @@
+import { makeStyles, Theme } from '@material-ui/core'
```

Comment:
```suggestion
import { makeStyles, Theme } from '@material-ui/core/styles'
```
would minimize bundle size